src/MaciVotingSetup.sol

```diff
@@ -92,16 +67,24 @@ contract MaciVotingSetup is PluginSetup {
         bytes memory _data
     ) external returns (address plugin, PreparedSetupData memory preparedSetupData) {
         (
-            SetupMACIParams memory setupMaciParams,
+            IMaciVoting.InitializationParams memory _params,
             GovernanceERC20.TokenSettings memory tokenSettings,
             GovernanceERC20.MintSettings memory mintSettings
         ) = abi.decode(
                 _data,
-                (SetupMACIParams, GovernanceERC20.TokenSettings, GovernanceERC20.MintSettings)
+                (
+                    IMaciVoting.InitializationParams,
+                    GovernanceERC20.TokenSettings,
+                    GovernanceERC20.MintSettings
+                )
             );
 
         address token = _deployToken(_dao, tokenSettings, mintSettings);
-        plugin = _deployPlugin(_dao, token, setupMaciParams);
+
+        _params.dao = IDAO(_dao);
```

Comment:
Nit: Could this just come from params? Instead of it coming from a dedicated function arg and params. Thinking here is to reduce configuration effort and prevent a mismatch due to the variable appearing twice

Reply:
ok let's leave it then